index.js

```diff
@@ -0,0 +1,105 @@
+import {
+  displayBooks,
+  displayForm,
+  contact,
+  displayContact,
+  showBooksOnly,
+  showOnlyContact,
+  showOnlyForm,
+} from "./modules/view.js";
+
+import { DateTime } from "./modules/luxon.js";
+
+const displayDate = document.querySelector(".date");
+
+displayDate.innerHTML = `${DateTime.local().toLocaleString(
+  DateTime.DATETIME_MED_WITH_SECONDS
+)}`;
+
+const form = document.querySelector("#form");
+let Author = document.querySelector(".Author");
+let Title = document.querySelector(".Title");
+let bookText = document.getElementById("book-text");
+
+class Book {
+  constructor() {}
+
+  static validateInputData = (title, author) => {
+    if (title.length < 1 || author.length < 1) {
+      return false;
+    }
+    return true;
+  };
+
+  save = (title, author) => {
+    let books = [];
+    if (localStorage.getItem("book") === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem("book"));
+    }
+    let book = { text1: title, text2: author };
```

Comment:
[OPTIONAL] Since you are not reassigning this variable, the proper declaration is const.